tests/Google/Ads/GoogleAds/Examples/Utils/ArgumentParserTest.php

```diff
@@ -97,7 +97,7 @@ public function testPassingRequiredAndOptionalArgumentsWhenOptionalArgumentMissi
     public function testPassingRequiredArgumentButMissingValue()
     {
         $this->expectOutputRegex('/Usage/');
-        $_SERVER['argv'] = [null, '--customerId', '--campaignId', 11111];
+        $_SERVER['argv'] = [null, '--campaignId', 11111, '--customerId'];
```

Comment:
This shouldn't be changed.
The test is aimed to test the case when customerId (which is required) is specified, but no value is given.

Reply:
You're right, but getopt changed its behavior between versions and now interprets '--campaignId' as the value for '--customerId', so to test the case when customerId is specified but no value is given this is correct.

Do you think we should add a specific test for the case when an example is invoked with something like `--customerId --campaignId 111111` or should we just leave it as it is, having examples fail with `request_error: Invalid customer ID '--campaignId'`?

Reply:
Thanks for confirming. Then, this already LGTM.